docs/data/charts/axis/CustomTickFrequency.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react';
+import Box from '@mui/material/Box';
+import { BarChartPro } from '@mui/x-charts-pro/BarChartPro';
+import { BarSeriesType, XAxis, YAxis, ZoomOptions } from '@mui/x-charts-pro/models';
+import { tickFrequencies } from '@mui/x-charts/utils';
+import alphabetStock from '../dataset/GOOGL.json';
+
+export default function CustomTickFrequency() {
+  return (
+    <Box sx={{ width: '100%' }}>
+      <BarChartPro
```

Comment:
This isn't shown in the preview. Do we want to fix that or move it outside the component?

Reply:
Not sure to get the point. I've shorten the code to let it appear in a preview